admin/server-timing.php

```diff
@@ -0,0 +1,131 @@
+<?php
+/**
+ * Server-Timing API admin integration file.
+ *
+ * @package performance-lab
+ */
+
+/**
+ * Adds the Server-Timing page to the Tools menu.
+ *
+ * @since n.e.x.t
+ */
+function perflab_add_server_timing_page() {
+	$hook_suffix = add_management_page(
+		__( 'Server-Timing', 'performance-lab' ),
+		__( 'Server-Timing', 'performance-lab' ),
+		'manage_options',
+		PERFLAB_SERVER_TIMING_SCREEN,
+		'perflab_render_server_timing_page'
+	);
+
+	// Add the following hooks only if the screen was successfully added.
+	if ( false !== $hook_suffix ) {
+		add_action( "load-{$hook_suffix}", 'perflab_load_server_timing_page' );
+	}
+
+	return $hook_suffix;
```

Comment:
What are you returning this for? This is hooked into an action right?

Reply:
Right, this would only be useful for test coverage, which I'm planning to add.